ci/build.sh

```diff
@@ -4,37 +4,44 @@
 set -e
```

Comment:
Nice of you that you want to format the script, but it was formatter with a different formatter already 😅 

I think we can remove these changes if you agree.

Reply:
Deleted the commit. Totally agree. Is it worth adding some pre-commit-config in the future? 😋

Reply:
It would make sense, yeah. although I'm not the most familiar with those configs. If you want, you can come up with something